tests/php/Model/LocaleTest.php

```diff
@@ -101,25 +102,25 @@ public function testGetBaseURLContainsDomainAndURLSegmentForNonDefaultLocale()
         // es_ES has a domain but is not the default locale for that domain
         $result = Locale::getByLocale('es_ES')->getBaseURL();
         $this->assertStringContainsString('fluent.es', $result, "Locale's domain is in the URL");
-        $this->assertStringContainsString('/es/', $result, 'URL segment for non-default locale is in the URL');
+        $this->assertStringEndsWith(Controller::normaliseTrailingSlash('/es/'), $result, 'URL segment for non-default locale is in the URL');
```

Comment:
Use `endsWith` instead of `contains` - it shouldn't just be arbitrarily _somewhere_ in the url, it should be specifically at the end.